psp-examples/swap/tests/swaps.ts

```diff
@@ -284,17 +344,24 @@ describe("Test swaps", () => {
       solMerkleTree: buyerUser.provider.solMerkleTree,
     });
 
+    /**
+     * Generates the system proof.
+     * The system proof proves the correct spending of input and creation of output utxos.
+     * Input utxos have to exists in the protocol state.
+     * Output utxos' asset amounts have to match the sums and assets of the input utxos.
+     */
     const systemProof = await getSystemProof({
       account: buyerUser.account,
       transaction: txParams,
       systemProofInputs: proofInputs,
     });
 
     /**
-     * Proves PSP logic
-     * returns proof and it's public inputs
+     * Generates the PSP proof.
+     * The PSP proof proves the PSP logic.
+     * In this case it enforces the constraints that an offer utxo
+     * can only be spent if a reward utxo exists for which the offer utxo data matches.
      */
```

Comment:
/**
 * Generates the proof for the Private Solana Program (PSP).
 * This proof enforces the constraints ensuring an offer UTXO can only be spent 
 * if a corresponding reward UTXO exists with matching offer UTXO data.
 */